src/basic_recipes/band.jl

```diff
@@ -54,6 +54,7 @@ function Makie.plot!(plot::Band)
     end
     attr = Attributes(plot)
     attr[:color] = meshcolor
+    attr[:alpha] = plot.alpha
```

Comment:
Ah, I'm not sure what the intention was here - this calls `Attributes` the constructor, which I _think_ makes sense, but there's an `attributes` method which gives `plot.attributes`.

I just realized this is really bad, because the transformation of `band` is not passed down to its `mesh` - this means that e.g. `translate!` on band doesn't work!  Will look into a more comprehensive fix for this.